pkg/plugin/README.md

```diff
@@ -0,0 +1,26 @@
+# Aethers Plugin Systems
+
+We have based our plugin system off of [hashicorps go-plugins](https://github.com/hashicorp/go-plugin/tree/main). This was mainly so we could detach the need for matching dependencies. We use gRPC for communication
+
+## Example Plugin
+
+There is an example plugin in the [example directory](./example/example.go) with comments on the moving parts. 
+
+## Install a Plugin
+
+// TODO //
+
+## Create a Docker Image with your Plugin
+
+// TODO //
+
+## Requirements for local setup
+
+as we use gRPC and protobuffers, if you want to make changes to the `.proto`
```

Comment:
Maybe an issue for adding a step in the pipeline that fails if there are proto changes and not buf generation?